aim/web/ui_v2/src/components/LineChart/styles.module.css

```diff
@@ -0,0 +1,54 @@
+.HoverLine {
```

Comment:
@KaroMourad can you please change this file name.
I think `LineChart.module.css` would be more helpful